README.md

```diff
@@ -6,3 +6,6 @@ Its main responsibilities are to provide:
 - getting/setting/validation functionality for identifiers for downstream processing and analysis
 - convenience functions for switching between long table and SummarizedExperiment data structures
 - core fitting functions
+
```

Comment:
Could you add free line between these points above - to better readability on site?
```
Its main responsibilities are to provide:

- getting/setting/validation functionality for identifiers for downstream processing and analysis

- convenience functions for switching between long table and SummarizedExperiment data structures

- core fitting functions
```

Reply:
done.

Reply:
done